client/src/components/profile/components/Camper.js

```diff
@@ -46,6 +47,26 @@ function joinArray(array) {
   });
 }
 
+function parseDate(joinDate) {
```

Comment:
```suggestion
function parseDate(joinDate) {
  joinDate = new Date(joinDate);
```
I *think* what's going on here is the Date gets serialised to a JSON string.  It looks like we have to recreate the Date manually.  https://weblog.west-wind.com/posts/2014/jan/06/javascript-json-date-parsing-and-real-dates discusses this, and the conclusion seems to be `new Date(dateString)` is fine.

Reply:
I haven't yet fixed `getSessionUser()`, but when I tested this in the Mongo Shell, there seems to be an issue during recreating the JSON date with `new Date()` - 

![Untitled91](https://user-images.githubusercontent.com/11348778/79038265-0f515f00-7ba6-11ea-9200-717f204a3c3e.png)

Reply:
Sorry for the delay!

So, what's happening here is that `stringify` is inserting quotes around the date string, i.e. `a[0] === '"'`, so `Date` does not understand it. If you parse it, it works:

```js
d = new Date(JSON.parse(a))
// ISODate("2018-10-26T12:52:31Z")
```